Src/PerformanceCollector/Filtering/DocumentStream.cs

```diff
@@ -0,0 +1,237 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse.Helpers;
+
+    /// <summary>
+    /// Document stream defines a stream of full telemetry documents that need to be collected and reported.
```

Comment:
it is not a stream as it's not inherited from `Stream`. Can you please clarify the comment defining what does stream means